src/docs/reference/modules/Placements/README.md

```diff
@@ -4,9 +4,9 @@ The placements module allows editors to add custom placement logic.
 
 ## General concepts
 
-Custom placements can be provided by themes and modules using a `placement.json` file.
+Custom placements can be provided by themes and modules using a `placement.json` file, which only affects the front-end shape, like `MyPart`on the themes and modules.
```

Comment:
you can use `placement.json` to also change placement of shapes in the backend.

Reply:
through my practice, using ThePart_Edit , I got the same issue  as
[](https://github.com/OrchardCMS/OrchardCore/issues/11840)

Reply:
`TitlePart_Edit` should work. At least it should work. If it does not, we should investigate why it does not instead.

Reply:
The `placement.json` of a front-end theme will not work, but it works on an admin theme. See https://github.com/OrchardCMS/OrchardCore/issues/11840#issuecomment-1152350608
I don't think this sentence clarifies that.